src/Nancy.Tests/Unit/Diagnostics/DefaultRequestTraceFactoryFixture.cs

```diff
@@ -62,10 +62,7 @@ public void Should_initialize_case_insensitive_items_collection_when_case_sensit
                 trace.Items.Add("foo", "bar");
 
                 // Then
-                Assert.DoesNotThrow(() =>
-                {
-                    var value = trace.Items["FOO"];
-                });
+                var value = trace.Items["FOO"];

```

Comment:
Record.Exception gets my vote.. hate the Assert.Throws and Assert.DoesNotThrow stuff.